src/Commands/IndexCategoriesCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Carbon\Carbon;
+use Rapidez\Core\Commands\InteractsWithElasticsearchCommand;
+use Rapidez\Core\Models\Store;
+
+class IndexCategoriesCommand extends InteractsWithElasticsearchCommand
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rapidez:index:categories {store? : Store ID from Magento}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Index the rapidez categories to elasticsearch';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): int
+    {
+        $storeModel = config('rapidez.models.store');
+        $categoryModel = config('rapidez.models.category');
+        $stores = $this->argument('store')
+        ? $storeModel::where('store_id', $this->argument('store'))->get()
+        : $storeModel::all();
+
+        foreach ($stores as $store) {
+            $this->line('Indexing categories for store: '.$store->name);
+            $this->setStore($store);
+
+            $categoryModelInstance = new $categoryModel();
+            $data = config('rapidez.models.category')::select($categoryModelInstance->qualifyColumns(['entity_id', 'name', 'url_path', 'children_count']))
```

Comment:
This is pretty static now, maybe add the possibility to change/extend this with for example an Eventy filter